### PR TITLE
Add support for Plank's config and theme files.

### PR DESCRIPTION
**Pull request by PerilousBooklet**

### CODE REVIEW

File: plugins/language_ini.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@ local syntax = require "core.syntax"
 
 syntax.add {
   name = "INI",
-  files = { "%.ini$", "%.inf$", "%.cfg$", "%.editorconfig$" },
+  files = { "%.ini$", "%.inf$", "%.cfg$", "%.editorconfig$", "%.theme$", "%.dockitem$" },
   comment = ';',
   patterns = {
     { pattern = ";.-\n", type = "comment" },
```
Comment (Guldoman) [inline plugins/language_ini.lua:10]: I know it's unrelated, but could you change this line and the next in the usual way?

**Review by Guldoman — COMMENTED** [review]
Needs a version bump.

File: plugins/language_ini.lua (hunk 1 above)
Comment (PerilousBooklet) [reply plugins/language_ini.lua:10]: What do you mean by `usual way` ?

Comment (Guldoman) [reply plugins/language_ini.lua:10]: > What do you mean by `usual way` ?

`.-\n` -> `.*`

> And for the version bump, is `0.1.1` fine ? It's a minor addition.

Yep.

**Pull request merged**